lib/DB/MySQL/Native.pm6

```diff
@@ -415,28 +415,26 @@ class DB::MySQL::Native is repr('CPointer')
     method init(DB::MySQL::Native:U: --> DB::MySQL::Native)
         is native(LIBMYSQL) is symbol('mysql_init') {}
 
-    multi method mysql_options(int32 $option, Blob $arg --> int32)
+    multi method mysql_options(int32 $option, CArray[uint8] $arg --> int32)
         is native(LIBMYSQL) is symbol('mysql_options') {}
-    multi method mysql_options(int32 $option, Pointer $arg --> int32)
+    multi method mysql_options(int32 $option, CArray[uint32] $arg --> int32)
         is native(LIBMYSQL) is symbol('mysql_options') {}
 
     multi method option(mysql-option $option, Str:D $arg)
     {
-        use NativeHelpers::Blob;
         my int32 $o = $option;
 
         $.check(
-          $.mysql_options( $o, pointer-to($arg.encode) )
+          $.mysql_options( $o, CArray[uint8].new($arg.encode) )
```

Comment:
I wonder why you can't just do `$.mysql_options($o, $arg.encode)` here? NativeCall does support Blobs: https://github.com/MoarVM/MoarVM/blob/master/src/core/nativecall_dyncall.c#L988
I rely on this support in Inline::Perl5: https://github.com/niner/Inline-Perl5/blob/master/lib/Inline/Perl5.pm6#L62 and https://github.com/niner/Inline-Perl5/blob/master/lib/Inline/Perl5/Interpreter.pm6#L116

Reply:
@niner - Every time I have attempted to use Blobs via NativeCall, I have encountered a SEGV. Given your recommendation, I may take a look at them again. Thanks.